### PR TITLE
[38][API] Implement listing keyword for user

### LINKED ISSUE #38: [API] As a user, I can get my keyword list
As a user, I can see the list of uploaded keywords.

## Acceptance criteria

The API should return a keyword list of a logged-in user

### PR DESCRIPTION
**Pull request by byhbt**
Issue: https://github.com/byhbt/kwtool/issues/38

## What happened

- [API] As a user, I can get my keyword list

## Insight

- Returns a list of keywords by logged-in user.
 
## Proof Of Work

Screen Shot 2022-02-21 at 10 55 10 PM

### CODE REVIEW

File: lib/kwtool_web/views/api/v1/keyword_view.ex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+defmodule KwtoolWeb.Api.V1.KeywordView do
+  use JSONAPI.View, type: "keywords"
+
+  def fields do
+    [
+      :id,
```
Comment (rosle) [inline lib/kwtool_web/views/api/v1/keyword_view.ex]: Not sure if we need this. It should already be there 🤔

File: test/kwtool/crawler/keywords_test.exs [hunk 2]
```diff
@@ -134,6 +134,27 @@ defmodule Kwtool.KeywordsTest do
     end
   end
 
+  describe "get_keywords_by_user/1" do
+    test "given the user has the keyword, returns a list of keyword" do
```
Comment (rosle) [inline test/kwtool/crawler/keywords_test.exs]:
```suggestion
    test "given the user has the keyword, returns a list of keywords" do
```

File: test/kwtool/crawler/keywords_test.exs [hunk 3]
```diff
@@ -134,6 +134,27 @@ defmodule Kwtool.KeywordsTest do
...
+      user = insert(:user)
+      keyword = insert(:keyword, user: user)
+
+      [keyword_in_db] = Keywords.get_keywords_by_user(user)
+
+      assert keyword_in_db.id == keyword.id
+      assert keyword_in_db.phrase == keyword.phrase
+    end
+
+    test "given the keyword does not belong to the user, returns an empty array" do
```
Comment (rosle) [inline test/kwtool/crawler/keywords_test.exs]:
```suggestion
    test "given the keyword does NOT belong to the user, returns an empty array" do
```

or 

```suggestion
    test "given the user does NOT have any keyword, returns an empty array" do
```

**Review by rosle — APPROVED** [review]
Just some small comments, the rest looks good to me 👍

**Review by hanam1ni — APPROVED** [review]

**Pull request merged**